test/net/sourceforge/kolmafia/request/GenericRequestTest.java

```diff
@@ -79,4 +79,20 @@ public void parseDesignerSweatpants(int expectedSweat) {
 
     assertThat("sweat", isSetTo(expectedSweat));
   }
+
+  @Test
+  public void doesNotProcessPostChoiceWhenNotInChoice() {
+    var cleanup =
+        new Cleanups(
+            withNextResponse(200, html("request/test_choice_whoops.html")),
```

Comment:
This is clearly a subset of a responseText. I guess that is OK.
I wonder if KoL can include other stuff - like events - which could affect where we want to look for this response during response processing.

Reply:
What do you mean?

Reply:
Oh you mean the test fixture? It's just what the DEBUG log reported for the request that I made

```
Retrieved: https://www.kingdomofloathing.com/choice.php?whichchoice=999&pwd&option=1&topper=3&lights=5&garland=1&gift=2
11 header fields
Field: :status = [200]
Field: cache-control = [no-store, no-cache, must-revalidate, post-check=0, pre-check=0]
Field: content-encoding = [gzip]
Field: content-type = [text/html; charset=UTF-8]
Field: date = [Fri, 29 Jul 2022 13:44:42 GMT]
Field: expires = [Thu, 19 Nov 1981 08:52:00 GMT]
Field: pragma = [no-cache]
Field: server = [nginx/1.8.1]
Field: set-cookie = [AWSALB=VmQejbPpsWYA/jkaikO/7b/JH+LVG6USP/AR5CIWzNTCLZCAitEP4gmcUNIvyBgFc/FrHW5mD4kEBySFjobtayus73VCQqsGtVjhe0N9j5IayY9H12rH4bBYX+v+; Expires=Fri, 05 Aug 2022 13:44:42 GMT; Path=/, AWSALBCORS=VmQejbPpsWYA/jkaikO/7b/JH+LVG6USP/AR5CIWzNTCLZCAitEP4gmcUNIvyBgFc/FrHW5mD4kEBySFjobtayus73VCQqsGtVjhe0N9j5IayY9H12rH4bBYX+v+; Expires=Fri, 05 Aug 2022 13:44:42 GMT; Path=/; SameSite=None; Secure]
Field: vary = [Accept-Encoding]
Field: x-powered-by = [PHP/5.3.29]

Retrieving server reply
ResponseText has 438 characters.
<center><table  width=95%  cellspacing=0 cellpadding=0><tr><td style="color: white;" align=center bgcolor=blue><b>Choice Adventure</b></td></tr><tr><td style="padding: 5px; border: 1px solid blue;"><center><table><tr><td><font face=Arial>Whoops!  You're not actually in a choice adventure.<p><Center><a href=main.php>Back to the Main Map</a></center></font></td></tr></table></center></td></tr><tr><td height=4></td></tr></table></center>
Processing results...
Returned: <center><table  width=95%  cellspacing=0 cellpadding=0><tr><td style="color: white;" align=center bgcolor=blue><b>Choice Adventure</b></td></tr><tr><td style="padding: 5px; border: 1px solid blue;"><center><table><tr><td><font face=Arial>Whoops!  You're not actually in a choice adventure.<p><Center><a href=main.php>Back to the Main Map</a></center></font></td></tr></table></center></td></tr><tr><td height=4></td></tr></table></center>

```

Reply:
So that WAS the entire HTML returned by KoL.
OK, cool!
I am used to seeing whole pages with "html" and "body" and so on.